core/templates/pages/thanks-page/thanks-page.component.spec.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for thanks page component.
+ */
+
+// TODO(#7222): Remove the following block of unnnecessary imports once
+// App.ts is upgraded to Angular 8.
+import { UpgradedServices } from 'services/UpgradedServices';
+// ^^^ This block is to be removed.
+
+require('pages/thanks-page/thanks-page.component.ts');
+
+describe('Thanks page', function() {
+  var ctrl = null;
+
+  beforeEach(angular.mock.module('oppia', function($provide) {
+    var ugs = new UpgradedServices();
+    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
+      $provide.value(key, value);
+    }
+  }));
+  beforeEach(angular.mock.inject(function($componentController) {
+    ctrl = $componentController('thanksPage');
+  }));
+
+  it('should set thanks image url when $onInit is called', function() {
+    ctrl.$onInit();
+    expect(ctrl.thanksImgUrl).toBe('/assets/images/general/donate.png');
```

Comment:
Seems like the wrong name for the image...

Reply:
I didn't understand what you meant

Reply:
The it block says "thanks image url" while the image URL says donate.png.... Wanted to make sure that that is intentional...

Reply:
The thanks image url refers to the property name `ctrl.thanksImgUrl`